client/landing/stepper/declarative-flow/site-migration-flow.ts

```diff
@@ -163,7 +165,13 @@ const siteMigration: Flow = {
 					if ( action === 'skip_platform_identification' || platform !== 'wordpress' ) {
 						return exitFlow(
 							addQueryArgs(
-								{ siteId, siteSlug, from, origin: STEPS.SITE_MIGRATION_IDENTIFY.slug },
+								{
+									siteId,
+									siteSlug,
+									from,
+									origin: STEPS.SITE_MIGRATION_IDENTIFY.slug,
+									backToFlow: `/${ FLOW_NAME }/${ STEPS.SITE_MIGRATION_IDENTIFY.slug }`,
+								},
 								'/setup/site-setup/importList'
```

Comment:
A thought for a future PR might be to add a constant for the flow name in `site-setup-flow.ts`. Then we could use that constant here.